c2corg_ui/static/partials/map/map.html

```diff
@@ -13,7 +13,12 @@
     </div>
   </div>
 
-  <button type="button" ng-show="mapCtrl.edit" ng-click="mapCtrl.resetFeature()"
-          class="btn btn-primary btn-xs map-reset-feature" translate>Reset feature</button>
+  <button type="button" ng-show="mapCtrl.canReset()" ng-click="mapCtrl.resetFeature()"
+          class="btn btn-primary btn-xs map-reset-feature"
+          title="{{ 'Reset geometry' | translate }}" translate>Reset geometry</button>
```

Comment:
I was expecting a comment about this. ;) *Feature* is a technical term that you can only understand if you have a background in GIS. I know that you want to keep the changes to the translation files to a minimum. But as a new translation has to be given for "Delete geometry" anyway, I thought it would be a good opportunity to get rid of this confusing string.